SoftWidgetAPI/.env.json

```diff
@@ -1,5 +1,13 @@
 {
   "LOGGER": true,
   "NODE_ENV": "development",
-  "PORT": 5000
+  "PORT": 5000,
+  "MONGO_URL": "mongodb://root:$123456root@ds125526.mlab.com:25526/htm-softwidgetapi-dev",
+  "MONGO_USER": "root",
+  "MONGO_PWD": "$123456root",
+  "MONGO_DB": "htm-softwidgetapi-dev",
+  "MONGO_URL_Test": "mongodb://test:$123456test@ds125526.mlab.com:25526/htm-softwidgetapi-test",
+  "MONGO_USER_TEST": "test",
+  "MONGO_PWD_TEST": "$123456test",
+  "MONGO_DB_TEST": "htm-softwidgetapi-test"
 }
```

Comment:
This file should be .env instead of .env.json

Reply:
done.
Should I keep these Test env vars?